apps/map/.env.local.example

```
@@ -1,51 +1,56 @@
-# Get a free API key: https://console.cloud.google.com/google/maps-apis/
-# Enable "Maps JavaScript API" for your project.
-NEXT_PUBLIC_GOOGLE_API_KEY=your-google-maps-api-key-here
-
-# -- Map app API key (for SSG authentication during `next build`) -------------
-# This authenticates in-process router calls during static generation.
-# Must match a key in the local DB — `pnpm db:seed:local` inserts "local-map-key".
-# Requires Docker + local DB running during `next build`.
-NEXT_PUBLIC_MAP_API_KEY=local-map-key
-
-NEXT_PUBLIC_GA_MEASUREMENT_ID=****
+NEXT_PUBLIC_GA_MEASUREMENT_ID=****  
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)